packages/twenty-website/src/app/components/user-guide/UserGuideTocComponent.tsx

```diff
@@ -0,0 +1,21 @@
+interface Heading {
+  id: string;
+  value: string;
+}
+
+const UserGuideTocComponent = ({ headings }: { headings: Heading[] }) => {
```

Comment:
<img width="773" alt="Screenshot 2024-02-18 at 18 40 25" src="https://github.com/twentyhq/twenty/assets/6399865/d0c340e1-9e5c-4af1-b212-1d4f93585cba">

Hide TOC in tablet mode?